src/instructlab/sdg/block.py

```diff
@@ -42,8 +43,13 @@ def _load_config(self, config_path: str) -> Union[Dict[str, Any], None]:
         """
         Load the configuration file for this block.
 
+        If the supplied configuration file is a relative path, it is assumed
+        to be part of this Python package.
+
         :param config_path: The path to the configuration file.
         :return: The loaded configuration.
         """
+        if not os.path.isabs(config_path):
+            config_path = os.path.join(self.ctx.sdg_base, config_path)
```

Comment:
a non-blocking improvement here would be to check if `config_path` exists prior to `open()` on the next line. If it doesn't exist, we could log a friendly message about it.

or maybe the pythonic way is to catch the exception from `open()` and log from there (and then re-raise it)

Reply:
Yep, similar to Dan's error handling work in instructlab/eval#61